chasm/lib/scheduler/invoker_tasks.go

```diff
@@ -140,6 +142,19 @@ func (e *InvokerExecuteTaskExecutor) Execute(
 				compiledSpec:       s.compiledSpec,
 			}
 
+			lcs, err := s.LastCompletionResult.Get(ctx)
```

Comment:
I see you call this last completion result and state interchangeably, consider settling on a single term.